src/sentry/tasks/integrations/migrate_repo.py

```diff
@@ -47,5 +48,5 @@ def migrate_repo(repo_id: int, integration_id: int, organization_id: int) -> Non
         from sentry.mediators.plugins import Migrator
 
         Migrator.run(
-            integration=integration, organization=Organization.objects.get(id=organization_id)
+            integration=integration, organization=organization_service.get(id=organization_id)
```

Comment:
Does the `Repository` query on line 16 of this file also need to be updated? It looks like this task will run on control silo, as it is spawned from the integration's `post_install` hook method.